compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -5350,7 +5350,7 @@ private BLangLiteral createSimpleLiteral(Node literal, SyntaxKind sign, boolean
                     value = textValue.replaceAll("[fd+]", "");
                     originalValue = textValue.replace("+", "");
                 } else {
-                    value = textValue;
+                    value = textValue.replaceAll("[fd+]", "");
```

Comment:
`1f f1 = 2;` 
In the above example, `1f` is a finite type. Right?
In L1353 we create `BLangFiniteTypeNode` and use `createSimpleLiteral(Node literal)` function to create literal value. Here we set the `isFiniteType` flag as false. Isn't it the issue for this?

Reply:
My bad. Will update the fix. Thanks.